static/app/data/controlsiloUrlPatterns.ts

```diff
@@ -4,23 +4,35 @@ const patterns: RegExp[] = [
   new RegExp('^remote/heroku/resources(?:/[^/]+)?$'),
   new RegExp('^remote/beacon/$'),
   new RegExp('^remote/newsletter/unsubscribe/$'),
+  new RegExp('^remote/github-copilot/oauth/$'),
+  new RegExp('^remote/github-secret-scanning/$'),
   new RegExp('^remote/github/marketplace/purchase/$'),
-  new RegExp('^remote/hubspot/webhook/$'),
   new RegExp('^remote/channel-provision/account/$'),
+  new RegExp('^remote/vercel/v1/installations/new/plans/$'),
+  new RegExp('^remote/vercel/v1/installations/[^/]+/plans/$'),
+  new RegExp('^remote/vercel/v1/installations/[^/]+/resources/[^/]+/plans/$'),
+  new RegExp('^remote/vercel/v1/products/[^/]+/plans/$'),
+  new RegExp('^remote/vercel/v1/installations/[^/]+/$'),
+  new RegExp('^remote/vercel/v1/installations/[^/]+/resources$'),
+  new RegExp('^remote/vercel/v1/installations/[^/]+/resources/[^/]+$'),
+  new RegExp('^remote/vercel/oauth$'),
```

Comment:
What's the deal with all this stuff

Reply:
This file can lag behind when changes are made in getsentry which is what these changes look like.